src/devices/Is31fl3731/Is31fl3731.cs

```diff
@@ -99,6 +99,28 @@ public void WritePixel(int x, int y, int brightness, bool enable, bool blink)
             WriteLedPwm(x, y, brightness);
         }
 
+        /// <summary>
+        /// Set value for all LEDs.
+        /// </summary>
+        public void WriteAllPixels(int[,] brightness)
```

Comment:
Any reason you use `int` here? If this was `byte[,]`, you could even convert it to a span without an actual copy.

Reply:
Consistent with `int this[int x, int y]`
Should this and similar members be changed to byte?

Reply:
In addition, the pixel arrangement of each specific LED board is different, and it is necessary to correspond the pixel coordinates to the actual data address. So the xy coordinate in the original code does not use byte, but int, probably for this reason

Reply:
I'm also wondering why the previous code used int instead of byte as the brightness value. This change may involve code changes for multiple other classes, and a separate PR may be required to make this change.

As for direct copying, I don't think it's feasible. Because the pixel layout of the actual LED board may not exactly correspond to the data address, a GetAddress method is required to obtain the actual address, resulting in no direct copy

Reply:
Ok, that's a different issue then. Let's leave it as-is for now.